frontend/src/app/components/home/home.component.html

```diff
@@ -7,7 +7,7 @@
         <div fxLayout="row" class="metrics-title">Raiden Explorer</div>
 
         <div fxLayout="row" class="metrics-subtitle">
-          Metrics for the Raiden <span class="testnet-label" *ngIf="!main">testnet</span> network
+          Metrics for the <span class="testnet-label">{{network}}</span> Raiden network
```

Comment:
Can we omit this when we're on mainnet. 

Nits:
- there would be two spaces before Raiden, can we avoid that
- Please capitalize network

Reply:
I will add an `*ngIf` so we can leave it empty on mainnet configuration and avoid displaying the text.
Give me some time to think how I could resolve the double space issue.

Reply:
I removed the second space but there is padding in the span so it should have a similar effect, can you verify that it is ok?